filters/auth/oidc_introspection.go

```diff
@@ -139,7 +139,12 @@ func (filter *oidcIntrospectionFilter) Request(ctx filters.FilterContext) {
 		return
 	}
 
-	sub := token.Claims["sub"].(string)
+	sub, ok := token.Claims["sub"].(string)
+	if !ok {
+		unauthorized(ctx, r.Host, invalidSub, "", "")
```

Comment:
Looking at the function signature
https://github.com/zalando/skipper/blob/e819c038999358a9a79514bf1de11f7132079369/filters/auth/auth.go#L136

used arguments do not look right.